# Empty Request to save the log being recorded

---
# Response
bool success
string filename
string err_str
